components/ClubPage/Header.js

```diff
@@ -12,8 +12,9 @@ const Wrapper = s.div`
   align-items: center;
 `
 
-const Header = ({ club, userInfo, favorites, style, updateFavorites }) => {
+const Header = ({ club, userInfo, favorites, style, updateFavorites, subscriptions, updateSubscriptions }) => {
```

Comment:
Are you using [Prettier](https://prettier.io/)? i think this should wrap to next line under those styles

Reply:
Ran `prettier` on the file and pushed!